### ISSUE TITLE
DismissAfter is hardcoded in notification stack

### ISSUE THREAD

**Issue by juliogarciag** [pburtchaell/react-notification#43]
When using the stack like this:

```jsx
const notifications = [
  {
    key: 1,
    message: "Some Message",
    dismissAfter: 10000
  }
]

<NotificationStack
  notifications={notifications} />
```

The `dismissAfter` property is always 2000. I found here:

https://github.com/pburtchaell/react-notification/blob/master/src/notificationStack.js#L23

that the `dismissAfter` property is not being honored when the property is passed to stacked notification and the 2000 values is hardcoded.

**Comment by pburtchaell** [comment]
Thanks for bringing this to my attention. I am happy to fix it but I will be unavailable to do so for the next few weeks—busy with work and returning to school. If you are willing to make a PR, that would be great! :smile:

**Comment by pburtchaell** [comment]
This is fixed in `react-notification@4.2.0`. Thanks to @sergiofm for the PR. :smile:
